viewer/viewer.js

```diff
@@ -8868,8 +8868,12 @@ function main () {
     internals.previousNodesStats.push(info.nodes);
   });
 
-  expireCheckAll();
-  setInterval(expireCheckAll, 60*1000);
+  var pcapWriteMethod = Config.get("pcapWriteMethod");
```

Comment:
Good idea, but I think these 6 lines will need to be moved to below loadPlugins for it to work

Reply:
Ah of course. I've done this in my latest commit.